lib/rules/no-duplicate-landmark-labels.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const Rule = require('./base');
+const AstNodeInfo = require('../helpers/ast-node-info');
+
+function createErrorMessage(element) {
+  return `Duplicate label on <${element}> detected. If two of the same landmark elements or landmark roles are found in the same template, ensure that they have a unique label (provided by the aria-label or aria-labelledby attribute).`;
+}
+
+const ROLE_LANDMARK_MAP = {
+  banner: 'header',
+  main: 'main',
+  complementary: 'aside',
+  form: 'form',
+  search: 'form',
+  navigation: 'nav',
+  contentinfo: 'footer',
+};
+
+const LANDMARK_ELEMENTS = new Set(['header', 'main', 'aside', 'form', 'main', 'nav', 'footer']);
```

Comment:
```suggestion
const LANDMARK_ELEMENTS = new Set(['header', 'main', 'aside', 'form', 'nav', 'footer']);
```

`main` is in this `Set` twice which seems to be a minor mistake. There are no consequences to this mistake because it still evaluates in Javascript to `[ "header", "main", "aside", "form", "nav", "footer" ]` due to JS stripping duplicate values from `Set`'s.

Alternatively, we could rely on Javascript enforcing uniqueness by converting the `ROLE_LANDMARK_MAP`'s values to a `Set` like so:

```js
const LANDMARK_ELEMENTS = new Set(Object.values(ROLE_LANDMARK_MAP));
```

This would better match the pattern on the next line using `Object.keys` as well.